hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -78,6 +78,8 @@ public final class OmKeyInfo extends WithParentObjectId {
    */
   private List<OzoneAcl> acls;
 
+  private boolean isHsync;
```

Comment:
Marginally related: In the design doc it mentioned `OmKeyInfo` should have a `clientId` field. Is that still planned?

Reply:
You are right.  The value `metadata.get(HSYNC_CLIENT_ID)` is the clientId.  We should just reuse it.  I also forget that.

Reply:
@szetszwo what if we just have isHsync() that returns the field? And therefore no need to add one extra field.

```
boolean isHsync() {
  return metadata.containsKey(OzoneConsts.HSYNC_CLIENT_ID);
}
```

Reply:
Sure, it is perfect!